resources/Qtmodels/AddComponentWindow.qml

```diff
@@ -28,315 +28,309 @@ ExpandingWindow {
         anchors.fill: parent
         padding: 0
 
-        Pane {
-            id: setupPane
-            height: childrenRect.height
-            width: childrenRect.width
+        StackLayout {
+            id: stack
```

Comment:
this is just my opinion, might be wrong or you might disagree with it, but i think we should call this something a bit more descriptive rather than just stack. Perhaps something like `addComponentStack`? The reason i say this is that its declarative so anything can use `stack` and it might become a bit hard to find from elsewhere in the qml, and could cause confusion if we use another StackLayout